app/presenters/c2_status_presenter/c2_status_presenter_factory.rb

```diff
@@ -0,0 +1,21 @@
+class C2StatusPresenterFactory
```

Comment:
I think that having this factory in the `presenters` dir is inconsistent with how we deal with our other factory objects. ALso, this is a factory rather than a presenter so I think it makes more sense to put it in `models` - thoughts?